src/hooks/useLoginUrl/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { deriv_urls } from '@site/src/utils/loginurls';
+import { useCallback } from 'react';
+
+const useLoginUrl = (language) => {
+  const getUrl = useCallback(() => {
+    const config_server_url = localStorage.getItem('config.server_url');
```

Comment:
on first time usage, the `config.server_url` and `config.app_id` is empty and the auth link is broken, can you please add default values for these? 
this is the generated auth link for me `https://oauth.deriv.com/oauth2/authorize?app_id=null&l=en`

Reply:
Fixed the initial auth